pontoon/base/models.py

```diff
@@ -494,15 +556,20 @@ def serialize(self):
             'name': self.name,
             'nplurals': self.nplurals,
             'plural_rule': self.plural_rule,
-            'cldr_plurals': self.cldr_plurals_list(),
+            'cldr_plurals': self.cldr_id_list(),
         }
 
-    def cldr_plurals_list(self):
+    def cldr_id_list(self):
         if self.cldr_plurals == '':
             return [1]
         else:
             return map(int, self.cldr_plurals.split(','))
 
+    def cldr_plurals_list(self):
```

Comment:
Yeah. I'd keep this for the next PR to avoid making this one even bigger.

Will mark all such suggestions with :hooray.